hack/after-build/update-generated-docs.sh

```diff
@@ -39,8 +39,10 @@ shopt -u dotglob
 
 kube::util::gen-analytics "${KUBE_ROOT}"
 
+KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
```

Comment:
Even though it's not necessary used (yet) outside of docs, it's still being set and defaulted in >1 place.  Definitely prefer to have a single include for that in a common place.